javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,103 @@
+/* Fetch movies from this api:
+
+Create an array of bad movies
+Creat an array of bad movies since year 2000 */
+function badMoviesRating() {
+    fetch("https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json")
+        .then(Response => Response.json())
+        .then(movies => {
+            const badMovies = movies.filter(movie => movie.rating <= 4)
+            console.log(badMovies)
+            badMoviesSince2000 = badMovies.filter(badMovies => badMovies.year >= 2000)
+            console.log(badMoviesSince2000)
+
+
+        })
+}
+badMoviesRating()
+//const moviejson
+//console.log(movies.json())
+
+function resolveTimer(resolveAfter) {
+    return new Promise((resolve) => {
+        setTimeout(() => {
+            resolve();
+        }, resolveAfter * 1000);
+    });
+}
+resolveTimer(5).then(() => {
+    console.log("I am called asynchronously")
+})
+
+function setTimeoutPromises(miliseconds) {
+    return new Promise((resolve, reject) => {
+        setTimeout(() => {
+            resolve();
+        }, miliseconds);
+    })
+}
+setTimeoutPromises(3000).then(() => {
+    console.log("Called after 3 seconds");
```

Comment:
would it be possible to have the `3 seconnds` somehow depend on the integer 3000 that is passed into the `setTimeoutPromises()` function